spec/factories/vm.rb

```diff
@@ -1,7 +1,12 @@
 FactoryGirl.define do
-  factory :vm_full, :class => "Ovirt::Vm" do
+  factory :vm, :class => "Ovirt::Vm" do
+    service { build(:service) }
+    initialize_with { new(service, {}) }
```

Comment:
Can you flip these two lines?  It's cleare to me when the intializer comes first before specific attributes.